pkg/features/kube_features.go

```diff
@@ -961,9 +963,9 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	AnyVolumeDataSource: {Default: true, PreRelease: featuregate.Beta}, // on by default in 1.24
 
-	AppArmor: {Default: true, PreRelease: featuregate.Beta},
+	AppArmor: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
I think there are more use of those feature gates than just util_test.go. Can you please delete those as well?

Reply:
I don't think we should remove that logic in this release. We should probably clean it up in the next release (1.32) as a post GA thing. 

@tallclair - what do you think?

Reply:
Discussed offline and kubernetes recommends that we remove all references to these in the code base as we are setting LockToDefault to true. See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md?plain=1

Reply:
Done.